pkg/osquery/runtime/runtime_test.go

```diff
@@ -1,34 +1,29 @@
-//go:build !windows
-// +build !windows
-
 package runtime
 
+// these tests have to be run as admin on windows
+
```

Comment:
Is it worth baking this into an `if` with a either an error or skip?

Reply:
I added it to the test main func, it just logs to stdout, the alternative was to added it to ever single test, which seemed tedious